resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -173,8 +173,15 @@ s3.delete.object.cancel=Cancel
 s3.delete.bucket_failed=Failed to delete bucket {0}
 s3.download.object.action=Download...
 s3.download.object.failed=Failed to download object {0}
-s3.download.object.description=Download
+s3.download.object.browse.title=Select Download Location...
+s3.download.object.browse.description.single=Select file or folder to save file to
+s3.download.object.browse.description.multiple=Select folder to save files to
 s3.download.object.progress=Downloading ''{0}''
+s3.download.object.conflict.description=File ''{0}'' already exists in directory ''{1}''
+s3.download.object.conflict.skip=Skip
+s3.download.object.conflict.overwrite=Overwrite
+s3.download.object.conflict.skip_rest=Skip All
```

Comment:
Skip all conflicts?

Reply:
Ya, I wasn't a fan oh JB (Skip for All).

Skip All Confclits sounds good to me